### ISSUE TITLE
baudrate 115201

### ISSUE THREAD

**Issue by Fuermohao** [micropython/micropython#4337]
![image](https://user-images.githubusercontent.com/18239246/49570898-cfa3b300-f972-11e8-8fe8-ca9ddb13d2ea.png)

why the default baudrate  is 115201 in ESP32?

**Comment by dpgeorge** [comment]
> is there something wrong?

No, that's just the accuracy at which the MCU can represent any given baudrate.  Some will be closer to the requested rate than others, some will be exact.  It shouldn't matter because there is room for error in the UART protocol.

**Comment by Fuermohao** [comment]
thanks! I got it.
